### PR TITLE
2023 07 mc editorial

### PR DESCRIPTION
**Pull request by michael-n-cooper**
Editorial review

### CODE REVIEW

File: guidelines/index.html [hunk 1]
```diff
@@ -236,15 +236,16 @@ <h2>Approaches to conformance</h2>
...
-					<li><strong>Assertions and procedures:</strong> <em>Assertions</em> are attributable statements by a person or organization that they followed a <em>procedure</em> to improve accessibility. Assertions are addressed in Section 3.4.1. </li>
+					<li><strong>Outcomes with tests</strong>: Outcomes are verifiable statements that allow testers to reliably determine if the content being evaluated satisfies the user needs identified in the Guideline. Outcomes are addressed in <a href="#outcomes">Section 3.3.1 Outcomes</a>. Tests are addressed in <a href="#testing-outcomes">Section 3.3.2 Testing Outcomes</a>. </li>
+					<li><strong>Assertions and procedures:</strong>
+						<em>Assertions</em> are attributable statements by a person or organization that they followed a <em>procedure</em> to improve accessibility. Assertions are addressed in <a href="#assertions">Section 3.4.1 Assertions</a>. </li>
```
Comment (rachaelbradley) [inline guidelines/index.html:241]: Is the <em> correct?

File: guidelines/index.html [hunk 2]
```diff
@@ -433,26 +434,26 @@ <h2>Assertions and procedures</h2>
...
+				<p>An assertion is a formal claim of fact, attributed to a person or organization. In WCAG 3, an [=assertion=] is an attributable and documented statement of fact regarding procedures practiced in the development and maintenance of the content or product to improve accessibility. </p>
 				</section>
 
 			<section>				
 				<h4>Using assertions</h4>
-				<p>Assertions may supplement methods in one or more outcomes. Assertions should only be used on outcomes and guidelines that allow assertions. Organizations can make an assertion that they followed a procedure to claim conformance. Results when testing assertions are true/false - the organization making the assertion either provided the reqeuired documentation or it did not.</p>
+				<p>Assertions may supplement [=methods=] in one or more outcomes. Assertions should only be used on outcomes and guidelines that allow assertions. Organizations can make an assertion that they followed a procedure to claim conformance. Results when testing assertions are true/false - the organization making the assertion either provided the reqeuired documentation or it did not.</p>
```
Comment (rachaelbradley) [inline guidelines/index.html:442]: Is the [= correct?

File: guidelines/index.html [hunk 3]
```diff
@@ -433,26 +434,26 @@ <h2>Assertions and procedures</h2>
...
 				<p>Examples of procedures that may be used during implementation might include:</p>
-					<ul><li>Training</li>
-					<li>FTE (Full Time Equivalent) assignments</li>
-					<li>Skills testing</li>
-					<li>Coordination and documentation of accessibility processes</li>
-					<li>Setting the priority for remediation</li></ul> 
+					<ul><li>Training,</li>
+					<li>FTE (Full Time Equivalent) assignments,</li>
+					<li>Skills testing,</li>
+					<li>Coordination and documentation of accessibility processes,</li>
```
Comment (rachaelbradley) [inline guidelines/index.html:450]: If we are creating full lists do we need to add "and" or "or"?

**Review by rachaelbradley — CHANGES_REQUESTED** [review]

File: guidelines/index.html (hunk 1 above)
Comment (michael-n-cooper) [reply guidelines/index.html:241]: I'm not clear on the question...

File: guidelines/index.html (hunk 2 above)
Comment (michael-n-cooper) [reply guidelines/index.html:442]: Yes, that's respec for "link to the definition"

File: guidelines/index.html (hunk 3 above)
Comment (michael-n-cooper) [reply guidelines/index.html]: Might as well, I added

File: guidelines/index.html (hunk 1 above)
Comment (rachaelbradley) [reply guidelines/index.html:241]: Ah, it applied the code. You'd added em tags to the Assertions and I hadn't seen it elsewhere. Is that the correct styling?

**Review by rachaelbradley — APPROVED** [review]

**Pull request merged**